fortran-inference/inference_scaling_imagenet.F90

```diff
@@ -48,10 +55,16 @@ program main
     )
 
 call init_client(client, rank, use_cluster, timing_unit)
-if (should_set .and. rank == 0) call set_model(client, device_type, num_devices, batch_size)
 
 model_key = "resnet_model"
+poll_model_code = client%poll_model(model_key, 200, 100, poll_model_bool)
+if (poll_model_code /= SRNoError) stop 'SR Error finding model'
+if (.not. poll_model_bool) stop 'Bool Error finding model'
```

Comment:
False on the boolean response indicates that the model wasn't found. A return code other than SRNoError means that something went wrong trying to execute the poll command